src/providers/export/tensorFlowRecords.ts

```diff
@@ -176,19 +176,17 @@ export class TFRecordsJsonExportProvider extends ExportProvider<ITFRecordsJsonEx
     }
 
     private updateAssetTagArrays(element: IAssetMetadata, imageInfo: IImageInfo) {
-        element.regions.filter((region) => (region.type === RegionType.Rectangle ||
-                                                   region.type === RegionType.Square) &&
-                                                   region.points.length === 2)
+        element.regions.filter((region) => region.boundingBox)
                                .forEach((region) => {
                                     region.tags.forEach((tag) => {
                                         const index = this.project.tags.map((pTag) => pTag.name).indexOf(tag.name);
 
                                         imageInfo.text.push(tag.name);
                                         imageInfo.label.push(index);
-                                        imageInfo.xmin.push(region.points[0].x);
-                                        imageInfo.ymin.push(region.points[0].y);
-                                        imageInfo.xmax.push(region.points[1].x);
-                                        imageInfo.ymax.push(region.points[1].y);
+                                        imageInfo.xmin.push(region.boundingBox.left);
+                                        imageInfo.ymin.push(region.boundingBox.top);
```

Comment:
Is this still accurate?  The top width and height values are the size of the box, not the x/y coordinates.  If you needed the bottom right point you would need `(boundingBox.left + boundingBox.width)` && `(boundingBox.top) + boundingBox.height)`